### DISCUSSION TITLE
No Alerts on SOC

### DISCUSSION THREAD

**Discussion by papabear-ed** [Security-Onion-Solutions/securityonion#2764] [Unsupported Versions]
Alerts quit showing up on the SOC. I had this issue start around Jan 22, 2021 after the server had been running since Jan 6, 2021. I updated it yesterday to version 2.3.21 and it began showing alerts for an hour and then stopped again. The storage is hovering around 89%. 
![so_no_alert](https://user-images.githubusercontent.com/40575809/106025276-6f55bb00-607d-11eb-9479-d8a1b15e93f0.PNG)
![so_no_alert2](https://user-images.githubusercontent.com/40575809/106025277-6fee5180-607d-11eb-8f8b-543d5c982b16.PNG)

**Comment by weslambert** [comment]
On your search node, do you see recent `ossec`, `ids`, etc indices when performing `curl -s- https://localhost:9200/_cat/indices | sort -n`. Is it just alerts, or all data that you are not seeing?

**Reply by papabear-ed** [reply]
When running `curl -s- https://localhost:9200/_cat/indices | sort -` it doesn't give any response back. I do not see ossec or ids alerts. I have reinstalled using the SO ISO. I have (1) distributed server as "Manager Only" and (1) as a distributed "sensor only". I can ssh into it properly through the setup and I can verify that the salt-key is active and the Salt-Master can push to the minions. 

I can view  events in `/nsm/suricata` on the "sensor node". Which verifies we are getting data / alerts for IDS.

**Reply by weslambert** [reply]
Looks like GH garbled the text when using the formatting:

Try:

curl -sk https://localhost:9200/_cat/indices | sort -n

**Reply by papabear-ed** [reply]
Thanks for helping me out.

The command doesn't prompt any return, as if executed. 

What data is being transferred over port 9200 that we are testing?

**Reply by rammyfzl** [reply]
I am having a similar issue where I am not receiving suricata alerts. And all files in _/nsm/suricata_ are 0 bytes.

**Reply by pappajohndd** [reply]
I have been fighting this for a week now. Redeployed everything several times. Can't get a sensor to work on centos or ubuntu. Was working great for a month then I lost the suricata container. Haven't been operational since.

**Reply by papabear-ed** [reply]
UPDATE:
I redeployed the "Manager Only" node as a Stand Alone and (2) other "Distributed, Sensor only" installs and it is working. It has been working for a week now. I also went with a minimal install of only Suricata, Kibana, and Grafana. All (3) nodes are running Ubuntu 18.04.

**Reply by pappajohndd** [reply]
no search node?

**Reply by launchdaemon** [reply]
> curl -sk https://localhost:9200/_cat/indices | sort -n

I am having a similar issue and this does not return anything for me.  `/nsm` is hovering around 90% usage I don't know if that is related but I can see files in `/nsm/suricata` are still being written.

**Reply by papabear-ed** [reply]
> 
> 
> no search node?

No, I do not use a search node.

**Reply by Courts3003** [reply]
I have exactly the same problem and exactly the same screens. So storage hovering at 89%, because the manager, search node and sensors have been working for about 20 days, no alerts being displayed (as above) and nothing on the Kibana display also (i.e. Just a 0). 

I am about to reinstall with 2.3.30 (I am presently on 2.3.21), but I do think that something unexpected is happening. I seems to happen when the "Automatic Purging" code kicks in because the storage is filling up (i.e. reaches 90%). 

I have 1 Manager, 1 Search Node, 2 Sensors running on various VMware Workstation Servers. Is there anything I can try in addition to the above suggested commands to help analyse and resolve the issue?

**Reply by rlg2019** [reply]
I'm having the same problem. No alerts on the dashboard. Data is being collected, but the dashboard doesn't show anything. The curl command displays a "missing authentication credentials for REST request" error. All services are running. It's a standalone system and has been fully updated with "soup."

**Reply by dougburks** [reply]
@rlg2019 This is an old discussion from over a year ago. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Reply by rlg2019** [reply]
Sorry, Doug. I only noticed the month, day. :-)

**Comment by meredib** [comment]
@edkernsjr; I just upgraded to 2.3.30 and it appears to have resolved the problem I was having similar to yours.  Granted I could not just do an in-line upgrade.  I had to build from scratch which was fine in my case because it was a new build.

**Comment by gbrodt** [comment]
Having a similar issue, using .100 & .120 Have you tried any of the newer versions?
